pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -615,6 +616,8 @@ def required_predicates(self) -> list[Predicate]:
             assert self.backend_info is not None
             preds.append(MaxNQubitsPredicate(self.backend_info.n_nodes))
             preds.append(MaxNClRegPredicate(cast(int, self.backend_info.n_cl_reg)))
+        if self.simulator_type == "stabilizer":
```

Comment:
Can we add a test or two showing that this is working?

Reply:
There's a related test [here](https://github.com/CQCL/pytket-quantinuum/blob/42cd71c7d581c6c94bd3cde759673ce73c18a3a1/tests/integration/backend_test.py#L599) which could be extended. (In fact, don't we expect that test to now throw an error from `process_circuit()`?)